README.md

```diff
@@ -101,29 +106,45 @@ namespace KontentAiModels
 }
 ```
 
-### Customizing models - Handling content element constraints
+### Customizing models - Extended delivery models
+Provides support to customize generated models based on content linked/subpages element constraints. This feature uses [Management SDK](https://github.com/kontent-ai/management-sdk-net) thus you'll need to provide api key as well.
 
-Currently, the generator is built on top of the Delivery API which doesn't provide information about content element constraints such as "Allowed Content Types" or "Limit number of items". In case you want your models to be more specific, this is the best practice on how to extend them:
+`KontentModelGenerator --projectid "<projectid>" -e true -k "<apikey>"`
 
+#### Extended delivery models example output
+Model is generated using structured model option ModularContent.
 Model.Generated.cs
 
 ```csharp
-public partial class Home
+public partial class Home : IContentItem
 {
-    public IEnumerable<object> LinkedContentItems { get; set; }
+    public const string SingleAllowedTypeSingleLinkedContentItemCodename = "single_allowed_type_single_linked_content_item";
+    public const string SingleAllowedTypeMultiLinkedContentItemsCodename = "single_allowed_type_multi_linked_content_items";
+    public const string MultiAllowedTypesSingleLinkedContentItemCodename = "multi_allowed_types_single_linked_content_item";
+    public const string MultiAllowedTypesMultiLinkedContentItemsCodename = "multi_allowed_types_multi_linked_content_items";
+
+    // Allowed Content Types == "Article" && Limit number of items == 1
```

Comment:
I would extend this comment for not exactly and at most.

Reply:
I think it might be better to introduce separate property